### ISSUE TITLE
Disabled UI a11y issues

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1177]
Legends, text and link within a "disabled UI" pattern fail colour contrast. Potentially just needs aria-disabled="true" adding to the `fieldset` (or `form`?). Also, links within the disabled ui can still be tabbed to / gain focus.

![image](https://user-images.githubusercontent.com/756393/77064249-0fab6f80-69d7-11ea-9a4f-2914af640c7c.png)

**Comment by Stanton** [comment]
The solution to this issue is two-fold, setting `aria-disabled` on the wrapping element, and stopping the disabled link from using a lighter text colour (as mentioned in #1169)

We've mentioned adding this to the `fieldset` or `form`, but adding to the disabled wrapper `div` works too, the WAI documentation seems to say any element can have this attribute:

----
https://www.w3.org/WAI/PF/aria/states_and_properties#aria-disabled

> The state of being disabled applies to the current element and all focusable descendant elements of the element on which the aria-disabled attribute is applied.

Characteristic | Value
-- | --
Used in roles: | All elements of the base markup
Value: | true/false
